ajax/libs/possum/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "possum",
+  "version": "1.0.4",
+  "description": "I am able. A State Machine.",
+  "babel": {
+    "presets": [
+      "es2015"
+    ]
+  },
+  "keywords": [
+    "hierarchical",
+    "state",
+    "machine",
+    "marsupial",
+    "fsm",
+    "finite"
+  ],
+  "author": "Mike Nichols <nichols.mike.s@gmail.com>",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mnichols/possum"
+  },
+  "license": "MIT",
+  "dependencies": {
+    "cuid": "^1.2.4",
+    "stampit": "^2.1.0"
+  },
+  "npmName": "possum",
+  "npmFileMap": [
+    {
+      "basePath": "/src",
```

Comment:
The basepath should be ```src``` not ```/src```.
And ```src``` is not a good basepath.
You can modify it to empty and modify the files to ```src/*```.

If you modify these fields, the filename should be modified too.